srcpkgs/discord/template

```diff
@@ -2,7 +2,7 @@
 # Originally created by Benjamin Hoffmeyer, modified for stable Discord
 
```

Comment:
Could you remove this newline and then solve all the resultant xlint warnings? For the license, use `license="custom:Proprietary"`

Reply:
@FirstZero:

> Could you remove this newline and then solve all the resultant xlint warnings?